scripts/prepdocslib/embeddings.py

```diff
@@ -180,13 +180,8 @@ def __init__(
         self.credential = credential
         self.organization = organization
 
-    async def create_embedding_arguments(self) -> dict[str, Any]:
```

Comment:
This was another change in Shakirs PR that looks like a bad merge, so I reverted it to what was in main.